engine/src/main/java/org/terasology/logic/players/FirstPersonHeldItemMountPointComponent.java

```diff
@@ -50,14 +50,28 @@
             0.0f, -0.05f, -0.2f, 1.0f
     );
 
+    /**
+     * If possible, make this object listen for controller pose upates.
+     */
     public void trySubscribeToControllerPoses() {
         vrProvider.getState().addControllerListener(this);
     }
 
+    /**
+     * A callback target for the controller listener. Not intended to be called manually.
+     * @param stateBefore - the state before the state change.
+     * @param stateAfter - the state after the state change.
+     * @param nController - the hand index, 0 for left and 1 for right.
```

Comment:
Actually, the upstream API provides up to 10 tracked devices at this time (and I think that's an arbitrary number), and some additional tracked devices are expected soon:
Main article:
http://www.roadtovr.com/valve-third-party-lighthouse-api-steamvr-tracking-htc-vive-dev-kit-license-royalty/
A tracked "puck" that you could put on the end of a baseball bat or something (lock up your cats).
http://www.roadtovr.com/htc-vive-tracker-enables-a-bevy-of-bats-guns-gloves-and-other-tracked-accessories/